romanesco/format/netcdf/binary_to_dataset.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "Binary to Dataset",
+    "inputs": [{"name": "input", "type": "netcdf", "format": "binary"}],
+    "outputs": [{"name": "output", "type": "netcdf", "format": "dataset"}],
```

Comment:
Format is subsidiary to type - both are always needed. That is, in UI's, API's etc the full format will show as something like "netcdf:dataset". Other types may also have the format "dataset" without conflict.